tests/test_bash_generation.sh

```diff
@@ -0,0 +1,7 @@
+cd ..
+python zppy/post.py -c tests/test_bash_generation.cfg
+ls test_output/post/scripts
+# https://stackoverflow.com/questions/2019857/diff-files-present-in-two-different-directories
+diff -bur test_output/post/scripts tests/expected_output
```

Comment:
Don't store expected output in the repo. Keep the expected output for the bash file/E3SM Diags/MPAS Analysis diffs in a folder on LCRC, like we do for E3SM Diags' complete run test.